cellpack/autopack/Analysis.py

```diff
@@ -678,10 +685,19 @@ def plot_occurence_distribution(self, ingr):
         self.simpleplot(
             range(len(occ)),
             occ,
-            self.figures_path / f"{ingr.name}_occurrence_{self.env.basename}.png",
+            self.figures_path
+            / f"{ingr.name}_occurrence_{self.env.basename}_lineplot.png",
             title_str=ingr.name,
             x_label="seed",
-            y_label="occurences",
+            y_label="occurrences",
+        )
+        self.histo(
```

Comment:
is there still a `self.histo`? It should be `self.env`

Reply:
The function name has been updated